lib/jbuild

```diff
@@ -80,4 +95,4 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
 (alias
  ((name runtest)
   (deps (channel_helper.exe))))
-|} flags coverage_rewriter flags coverage_rewriter flags
+|} runtime_coverage_enabled flags coverage_dep coverage_rewriter flags coverage_rewriter flags
```

Comment:
These are a long way from the `%s` tokens that refer to them, so it's inconvenient to work out which refers to which.
I wonder if there might be a nicer way to handle this.

Reply:
We could build all the chunks separately and join them at the end instead of creating all the string in one go. We can't do much better than that.

Reply:
We could replace each `%s` with something like
`|} ^ runtime_coverage_enabled ^ {|`
`|} ^ flags ^ {|`
and so on.

I not that using these variables at all depends on a jbuild feature which might be removed in a future release:
https://jbuilder.readthedocs.io/en/latest/jbuild.html#ocaml-syntax

Reply:
Our use is far from complex, I would not be too worried about that. Also, aside for coverage, we should start using much less of it once we can move to omp-based ppx rewriters.

Anyways, I would rather sprintf the stanzas separately and ksprintf their concatenation. I always found the  _^_^_ string concatenation hard to read